src/figuresClasses.ts

```diff
@@ -1,11 +1,82 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
Consider using the `Color` type alias instead of repeating the color options. This will ensure consistency and make it easier to update the color options in the future.